common/djangoapps/student/auth.py

```diff
@@ -20,7 +20,7 @@
 # In addition to the above, one is always allowed to "demote" oneself to a lower role within a course, or remove oneself
 
 
-def has_access(user, role):
+def has_role(user, role):
```

Comment:
nit: Could this variable be called something more readable like `has_access_to_role` or something of the sort?